apps/idos-enclave/src/pages/methods/Passkey.tsx

```diff
@@ -64,61 +64,73 @@ export default class Passkey extends React.Component<PasskeyProps> {
     let credentialId;
     let password;
 
-    const { store } = this.props;
+    const { store, mode } = this.props;
 
     const storedCredentialId = store.get("credential-id");
 
+    const credentialRequestWithoutId = {
+      publicKey: {
+        challenge: crypto.getRandomValues(new Uint8Array(10)),
+      },
+    };
+
     if (storedCredentialId) {
-      const credentialRequest = {
+      const credentialRequestWithId = {
+        ...credentialRequestWithoutId,
```

Comment:
I'd remove this line, since we override it right after.

Reply:
That is true, but this spread in specific feels useless, since there's only `publicKey` in `credentialRequestWithoutId` and we set it manually on the next line.